modules/form-builder/main/index.coffee

```diff
@@ -262,6 +281,21 @@ class Form extends hx.EventEmitter
       else
         !!prop.hidden
 
+  disabled: (property, disabled) ->
+    if hx.isArray(property)
+      res = property.map (prop) => @disabled(prop, disabled)
+      if disabled? then this else res
```

Comment:
I just went with what the hidden function already did and that supports an array - It's pretty much the same function so if it needs changing then they both do.